js/components/dataset/card.vue

```diff
@@ -92,7 +92,11 @@ import DatasetFilters from 'components/dataset/filters';
 import placeholders from 'helpers/placeholders';
 import config from 'config';
 
-const MASK = 'id,title,description,spatial{zones,granularity},metrics,organization,frequency,temporal_coverage';
+const MASK = [
```

Comment:
As we discussed – and this is subjective – I know that `mask` actually refers to `X-Fields`in the API meta. I would therefore have called it _XFIELDS_ or _FIELDS_ to make it more uniform.
An extra opinion would be welcome.